src/src/Helpers/Hardware.cpp

```diff
@@ -623,22 +623,22 @@ uint8_t temprature_sens_read();
 
 float getInternalTemperature() {
   float temperature = -273.15f; // Inprobable value
```

Comment:
Improbable, not inprobable

Reply:
Still doubting if it should return -1 instead of 0 Kelvin, as -1 also is quite improbable for a CPU 🤔

Reply:
Unless you have the unit mounted outside